src/host/snapshot.h

```diff
@@ -57,7 +57,7 @@ namespace asynchost
       if (fs::exists(full_snapshot_path))
       {
         throw std::logic_error(fmt::format(
-          "Cannot write snapshot at {} since file already exists: {}",
+          "Error: Cannot write snapshot at {} since file already exists: {}",
```

Comment:
We don't consistently prefix our logic_errors with "Error:", I'm not sure we should, but I think we shouldn't do it just for some of them.